pkg/kn/commands/plugin/plugin_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2018 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/knative/client/pkg/kn/commands"
+	"github.com/spf13/cobra"
+	"gotest.tools/assert"
+)
+
+const PluginCommandUsage = `Provides utilities for interacting with kn plugins.
```

Comment:
s/interacting with/managing/

Reply:
Well right now it’s interaction since there is not management features but I won’t dwell on what term to use... Let’s see if others want to chime in ... otherwise, I can do managing.